superset/views/core.py

```diff
@@ -2810,7 +2810,7 @@ def schemas_access_for_file_upload(self) -> FlaskResponse:
         get the schema access control settings for file upload in this database
         """
         if not request.args.get("db_id"):
-            return json_error_response("No database is allowed for your csv upload")
+            return json_error_response("No database is allowed for your file upload")
```

Comment:
Are you sure this is not being used when uploading a CSV as well?
@rusackas do you know if this API is being called from multiple places?

Reply:
@diegomedina248 

This API is used when the **csv** && **excel** && **columar** file is uploaded.

Reply:
I think this is fine. Approving the PR, but I'll let @yousoph merge it if she agrees, or leave a comment if she doesn't ;)